docs/modules/components/pages/processors/ollama_chat.adoc

```diff
@@ -123,6 +133,170 @@ This field supports xref:configuration:interpolation.adoc#bloblang-queries[inter
 *Type*: `string`
 
 
+=== `response_format`
+
+The response format of generated type, the model must also be prompted to output the appropriate response type.
```

Comment:
```suggestion
The format of the response that the Ollama model generates. For example, `text`. The `prompt` field value must match the `response format` to output the appropriate response type.
```
I'm not sure if this is what you mean? What would you need to include as prompt to generate a `text` response?